framework/PortalFrame/PortalFrame.css

```diff
@@ -38,3 +38,27 @@
         height:100vh;
     }
 }
+
+.account {
+	height:50px;
+	width:250px;
+	text-align: right !important;
+	position:fixed;
+	top:17px;
+	right:75px;
+	font-family:Nunito, sans-serif;
+	font-size:17px;
+	font-style:normal;
+	font-weight:600;
+	color:#FAFAFA;      
+}
+
+@media screen and (max-width: 600px) {
+    .mainarea {
+		position:absolute;
+		left:0px;
+		width:100vw;
+		height:calc( 100vh - 44px );
+		min-height:calc( 100vh - 44px );
+	}	
+}
```

Comment:
This media query is no longer necessary. There's a media query to handle the `mainarea` for smaller screens, right above this code-block in the same file. 
Also the `height` and `width` property doesn't respect the presence(and state) of drawer menu which is properly handled in the other media query.
So either properly set the `height` and `width` property respecting the state of drawer menu or simply removing this block would suffice for now.

Reply:
Thanks, removed.